src/components/DateInput.jsx

```diff
@@ -136,14 +157,12 @@ export default class DateInput extends React.Component {
           disabled={disabled}
           readOnly={isTouch}
           required={required}
-          aria-describedby={screenReaderMessage && screenReaderMessageId}
```

Comment:
if you were running into issues before where this was coming back as '' and still adding the attribute, then you might consider having the default value of `screenReaderMessage` as `null` or `undefined` so that this works